src/business-logic/services/test/rundown-timeline-service.spec.ts

```diff
@@ -55,6 +56,22 @@ describe(RundownTimelineService.name, () => {
 
       await expect(() => testee.deleteRundown(rundown.id)).rejects.toThrow(ActiveRundownException)
     })
+
+    it('throws an exception, when trying to active a rundown when there is another already activated rundown', async () => {
+      const mockRundownRepository: RundownRepository = mock<RundownRepository>()
+
+      const activeBasicRundown: Rundown = EntityMockFactory.createRundown({ isRundownActive: true })
+
+      const basicRundownArray: Rundown[] = [activeBasicRundown];
+
+      const rundownToActivate: Rundown = EntityMockFactory.createRundown({isRundownActive: false})
+
+      when(mockRundownRepository.getBasicRundowns()).thenResolve(basicRundownArray)
+
+      const testee: RundownTimelineService = createTestee({rundownRepository: instance(mockRundownRepository)})
+
+      await expect(() => testee.activateRundown(rundownToActivate.name)).rejects.toThrow(AlreadyActivatedException)
+    })
```

Comment:
You can use newlines to group the test into different logical sections, like in the code snippet below, with a setup section, a section where you apply the functionality to test and an assertion section.
I have taken the liberty to move the variable instantiation of the mocked RundownRepository down to right before the `when` statement to have the reading flow match the execution flow as close as possible.
I have also moved the expression to test out into the variable  `result` to make that section explicit.

```typescript
    it('throws an exception, when trying to active a rundown when there is another already activated rundown', async () => {
      const activeBasicRundown: Rundown = EntityMockFactory.createRundown({ isRundownActive: true })
      const basicRundownArray: Rundown[] = [activeBasicRundown];
      const rundownToActivate: Rundown = EntityMockFactory.createRundown({isRundownActive: false})
      const mockedRundownRepository: RundownRepository = mock<RundownRepository>()
      when(mockedRundownRepository.getBasicRundowns()).thenResolve(basicRundownArray)
      const testee: RundownTimelineService = createTestee({rundownRepository: instance(mockedRundownRepository)})
      
      const result: () => Promise<void> = () => testee.activateRundown(rundownToActivate.name)
      
      await expect(result).rejects.toThrow(AlreadyActivatedException)
    })
```